tests/models/test_base_election_model.py

```diff
@@ -300,3 +299,57 @@ def test_get_aggregate_predictions(va_governor_precinct_data):
         + df2.groupby("county_classification").sum().reset_index(drop=False)[f"pred_{estimand}"].values[0]
         == df4[f"pred_{estimand}"].values[0]
     )
+
+
+def test_compute_lambda_under_one():
+    """
+    Test/view computing lambda
+    """
+    lambda_ = [0.01, 0.05, 0.99, 0.56]
+    model_settings = {"features": ["b"]}
+    model = BaseElectionModel.BaseElectionModel(model_settings)
+    df_X = pd.DataFrame(
+        {
+            "residuals_a": [1, 2, 3, 4],
+            "total_voters_a": [4, 2, 9, 5],
+            "last_election_results_a": [5, 1, 4, 2],
+            "results_a": [5, 2, 8, 0],
+            "results_b": [0, 6, 2, 1],
+            "baseline_a": [9, 2, 4, 5],
+            "baseline_b": [9, 2, 4, 5],
+            "a": [2, 2, 3, 7],
+            "b": [2, 3, 4, 5],
+        }
+    )
+
+    new_lambda, avg_MAPE = model.compute_lambda(df_X, lambda_, "a")
+
+    assert new_lambda == 0.01
+    assert avg_MAPE == 0.625
```

Comment:
kinda curious where you got the 0.625 from?

Reply:
Initially, I had a bug where the average_MAPE was not consistent with the chosen lambda so I used that check during testing. I can remove it if needed!

Reply:
no, it's good to have. but like did you hand compute it?

Reply:
Yes, checked it by hand for 0.01

Reply:
amazing! do you mind just adding a comment there that you did that (so we don't get confused about where that 0.625 comes from later on?